corehq/apps/app_manager/models.py

```diff
@@ -430,6 +465,18 @@ def all_property_names(self):
     def count_subcases_per_repeat_context(self):
         return Counter([action.repeat_context for action in self.subcases])
 
+    @classmethod
+    def wrap(cls, data):
+        if 'update_case' in data and 'update' in data['update_case']:
```

Comment:
Hi @orangejenny , I'm trying to enable the code in this wrap only when a toggle is enabled on the relevant so that we can put this under a feature release toggle, but I'm having trouble finding a way to get the appropriate domain inside of this class. Do you have any ideas?

Reply:
It'll be tough to check a toggle in this part of the code, when you're still working with the raw json doc, you don't even have the class yet. Usually we flag closer to the UI level. For this feature, I'd think you could just flag the HTML for the checkboxes, since that's the user-facing part. Would that work?

Reply:
I was worried that without a toggle the models wouldn't be backwards compatible. I forgot that we need them to be backwards compatible during a migration anyways. And upon investigation, they actually are backwards compatible--prior bugs had made me thing that they weren't but I'm realizing those bugs could actually be unrelated. So I think you're right, I can just focus on UI (and maybe XML). Thanks Jenny.